nandtool/nandtool.c

```diff
@@ -177,6 +177,91 @@ static int nandtool_check(void)
 }
 
 
+static int nandtool_dump(const char *outpath, unsigned int start, unsigned int nblocks, int oob)
+{
+	flashsrv_info_t *info;
+	int ret;
+	size_t blocksz, partsz, pagesz;
+	size_t addr, endaddr;
+	uint8_t *buf;
+	int outfd;
+
+	outfd = open(outpath, O_RDWR | O_CREAT);
+	if (outfd < 0) {
+		fprintf(stderr, "nandtool: Fail to open file: %s\n", outpath);
+		return -EINVAL;
+	}
+
+	info = flashmng_info(nandtool_common.oid);
```

Comment:
not directly related to this PR but this function returns pointer to global object which shouldn't be changed by the caller (so, `const`)

Reply:
I will change the `info` pointer type to `const` at least here.